### PR TITLE
Geocode facility list items

### PR DESCRIPTION
**Pull request by kellyi**
## Overview

This PR adds a pipeline for geocoding PARSED facility list items into GEOCODED facility list items.

First commit adds a `geocoded_address` field to the base modal via an update to the 0001 migration.

Second commit

- adds the requests library
- creates some utility functions for doing geocoding
- creates a `geocode_facility_list_item` function for changing the item's state to GEOCODED if geocoding was successful and ERROR if it wasn't
- adds some tests for the pipeline

Connects #77

## Testing

- get this branch, `reset_schema` and `./scripts/manage migrate`
- `./scripts/test` to verify that the new pipeline is working as expected

### CODE REVIEW

**Comment by jwalgran** [comment]
Looking at this now

File: src/django/api/libs.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+from django.conf import settings
```
Comment (jwalgran) [inline src/django/api/libs.py]: Why `libs.py` instead of something like `geocoding.py`?

File: src/django/api/tests.py [hunk 2]
```diff
@@ -259,3 +265,98 @@ def test_generated_token_is_not_deleted_on_logout(self):
...
+
+    def test_unparsed_item_raises_error(self):
+        facility_list = FacilityList(header='address,country,name')
+        item = FacilityListItem(
+            raw_data='1400 JFK Blvd, Philly,us,Shirts!',
+            facility_list=facility_list
+        )
+
+        with self.assertRaises(ValueError) as cm:
+            geocode_facility_list_item(item)
```
Comment (jwalgran) [inline src/django/api/tests.py]: Today I learned (or possibly relearned) that the binding of a `with` block stays in scope after the block completes. Good to know.

File: src/django/api/libs.py (hunk 1 above)
Comment (kellyi) [reply src/django/api/libs.py]: No real reason, although `geocoding.py` is probably a better name. Will update!

Comment (kellyi) [reply src/django/api/libs.py]: I updated this and force pushed an amended commit.

File: src/django/api/tests.py (hunk 2 above)
Comment (kellyi) [reply src/django/api/tests.py]: I saw this in the Python tests documentation, which seemed like a nice way to test the type and result of an Exception!

**Comment by kellyi** [comment]
Thanks for your help with this! I'm going to merge.

**Pull request merged**